Uno.Gallery/Uno.Gallery.UITest/Given_Card_01.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Uno.UITest.Helpers.Queries;
+using Uno.UITests.Helpers;
+
+namespace Uno.Gallery.UITests
+{
+	public class Given_Card_01 : TestBase
+	{
+
+		/*This function is to test the Unchecked option  in checkbox for material */
+		[Test]
+		public void When_01_OutlinedCardClick()
+		{
+			NavigateToSample("Card", "Material");
+
+			TakeScreenshot("Before Click");
+			var outlinedCardClick = new QueryEx(x => x.All().Marked("Outlined_Card"));
+			//App.ScrollDownTo("Outlined_Card");
```

Comment:
`//App.ScrollDownTo("Outlined_Card");` still needed ?
@sakshi173  please make sure to verify all the comments like this from the entire PR. If still needed add the code if not please remove it.